app/components/migration/participant_profile_component.html.erb

```diff
@@ -40,11 +40,11 @@
     else
       list.with_row do |row|
         row.with_key(text: "Mentor completion date")
-        row.with_value(text: participant_profile.mentor_completion_date)
+        row.with_value(text: participant_profile.mentor_became_ineligible_for_funding_on)
```

Comment:
@tonyheadford just wanted to double check that these do need to be updated?

Let me know if there is anything else I should consider.

Reply:
No these are models connected to ECF1 so we shouldn't change these, thanks for checking 👍

Reply:
Nice one - i've removed. Thank you!